modules/covid_vaccine/spec/factories/registration_submissions.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
-  factory :covid_vaccine_registration_submission, class: 'CovidVaccine::V0::RegistrationSubmission' do
+  factory :covid_vax_registration, class: 'CovidVaccine::V0::RegistrationSubmission' do
```

Comment:
Is there a specific reason for renaming this from covid_vaccine_registration_submission?